stubs/tests/ResourceTest.php

```diff
@@ -23,14 +22,17 @@ public function test_users_can_create_resources()
         $response = $this->actingAs($user)->get(localized_route('resources.create'));
         $response->assertStatus(200);
 
-        $response = $this->actingAs($user)->post(localized_route('resources.create'), [
-            'user_id' => $user->id,
-            'title' => 'Test resource',
-            'language' => 'en',
-            'summary' => 'This is my resource.',
-        ]);
+        $response = $this->actingAs($user)->post(localized_route('resources.create'), 
+            Resource::factory()->create([
```

Comment:
Calling `Resource::factory` actually creates a new resource, but what the POST request here needs is the array of data. So reverting back to the previous code but removing the `language` attribute should work.

Reply:
You can check to see if the request was successful by asserting the following:

```php
$response->assertSessionHasNoErrors();
```